vcbuild.bat

```diff
@@ -108,44 +108,102 @@ if defined NIGHTLY set TAG=nightly-%NIGHTLY%
 
 @rem Set environment for msbuild
 
+@rem Look for Visual Studio 2015
+echo Looking for Visual Studio 2015
+if not defined VS140COMNTOOLS goto vc-set-2013
+if not exist "%VS140COMNTOOLS%\..\..\vc\vcvarsall.bat" goto vc-set-2013
+echo Found Visual Studio 2015
+if defined msi (
+  echo Looking for WiX installation for Visual Studio 2015...
+  if not exist "%WIX%\SDK\VS2015" (
+    echo Failed to find WiX install for Visual Studio 2015
+    echo VS2015 support for WiX is only present starting at version 3.10
+    goto vc-set-2013
+  )
+)
+if "%VCVARS_VER%" NEQ "140" (
+  call "%VS140COMNTOOLS%\..\..\vc\vcvarsall.bat"
+  SET VCVARS_VER=140
+)
+if not defined VCINSTALLDIR goto msbuild-not-found
```

Comment:
master and v0.12 diverged on this. I guess which strategy is better depends on whether VCINSTALLDIR not being set mean an error in vcvarsall.bat, or a VS installation without VC++. I would err on the side of trying the next version.